experiments/Atmos_GCM/heldsuarez.jl

```diff
@@ -1,61 +1,89 @@
+using Distributions: Uniform
 using LinearAlgebra
 using StaticArrays
+using Random: rand
 using Test
 
 using CLIMA
 using CLIMA.Atmos
 using CLIMA.GenericCallbacks
 using CLIMA.ODESolvers
+using CLIMA.ColumnwiseLUSolver: ManyColumnLU
 using CLIMA.Mesh.Filters
 using CLIMA.Mesh.Grids
 using CLIMA.MoistThermodynamics
 using CLIMA.PlanetParameters
 using CLIMA.VariableTemplates
 
-const p_ground = MSLP
-const T_initial = 255
-const domain_height = 30e3
 
-function init_heldsuarez!(bl, state, aux, coords, t)
-    global p_ground, T_initial
-
-    FT = eltype(state)
+const pressure_ground = MSLP
+const T_init = 300.0 # unit: Kelvin 
+const domain_height = 30e3 # unit: meters
 
-    # TODO: change to altitude function
-    r = norm(coords, 2)
-    h = r - FT(planet_radius)
-    # h = altitude(bl.orientation, aux)
 
-    scale_height = R_d * FT(T_initial) / grav
-    p            = FT(p_ground) * exp(-h / scale_height)
-
-    state.ρ  = air_density(FT(T_initial), p)
+function init_heldsuarez!(bl, state, aux, coords, t)
+    FT            = eltype(state)
+    pressure_sfc  = FT(MSLP)
+    temp_init     = FT(300.0)
+    radius        = FT(planet_radius)
+
+    # Initialize the state variables of the model
+    height = norm(coords, 2) - radius #TODO: altitude(bl.orientation, aux)
+    scale_height = R_d * temp_init / grav
+    pressure = pressure_sfc * exp(-height / scale_height)
+
+    rnd      = FT(1.0 + rand(Uniform(-1e-3, 1e-3)))
+    state.ρ  = rnd * air_density(temp_init, pressure)
     state.ρu = SVector{3, FT}(0, 0, 0)
-    state.ρe = state.ρ * (internal_energy(FT(T_initial)) + aux.orientation.Φ)
+    state.ρe = state.ρ * (internal_energy(temp_init) + aux.orientation.Φ)
 
-    return nothing
+    nothing
 end
 
-function config_heldsuarez(FT, N, resolution)
-    global T_initial, domain_height
-
-    ref_state = HydrostaticState(IsothermalProfile(FT(T_initial)), FT(0))
-    model = AtmosModel{FT}(AtmosGCMConfiguration;
-                           ref_state  = ref_state,
-                           turbulence = ConstantViscosityWithDivergence(FT(0)),
-                           moisture   = DryModel(),
-                           source     = (Gravity(), Coriolis(), held_suarez_forcing!),
-                           init_state = init_heldsuarez!)
-    config = CLIMA.Atmos_GCM_Configuration("HeldSuarez", N, resolution,
-                                           FT(domain_height),
-                                           init_heldsuarez!;
-                                           model = model)
+
+function config_heldsuarez(FT, poly_order, resolution)
+    name          = "HeldSuarez"
+    domain_height = FT(30e3)
+    T_ref         = FT(300.0)
+    Rh_ref        = FT(0.0)
+    turb_visc     = FT(0.0)
+    
+    # Rayleigh sponge 
+    zsponge = FT(15e3)
+    τ_relax = FT(0.5 / 60 / 60 / 24)
```

Comment:
```suggestion
    τ_relax = FT(0.5 / day)
```
(We used to have `day` (seconds per day) defined in `PlanetParameters`. I'm not sure we still have it, but we want something like this here.)